examples/complex_fluids/ex2/InterpolationUtilities.h

```diff
@@ -36,6 +33,10 @@
 
 #include <vector>
 
+#include <ibamr/app_namespaces.h>
+
+#include "ibamr/namespaces.h"
```

Comment:
Do we need both `app_namespaces.h` and `namespaces.h`?

Reply:
No - I'll fix that.